4.4.0/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+export JAVA_OPTIONS="${JAVA_OPTS} ${GN_CONFIG_PROPERTIES}"
+
+GN_BASE_DIR=/opt/geonetwork
+
+if ! command -v -- "$1" >/dev/null 2>&1 ; then
+	set -- java -jar "$JETTY_HOME/start.jar" "$@"
+fi
+
+if [[ "$1" = jetty.sh ]] || [[ $(expr "$*" : 'java .*/start\.jar.*$') != 0 ]]; then
+    # Customize context path
+    if [ ! -f "{$JETTY_BASE}/webapps/geonetwork.xml" ]; then
```

Comment:
There is an error reading the `JETTY_BASE` environment variable. It should be:
```suggestion
    if [ ! -f "${JETTY_BASE}/webapps/geonetwork.xml" ]; then
```